biohansel/subtype/qc/checks.py

```diff
@@ -4,10 +4,10 @@
 
 import pandas as pd
 
-from ..qc.const import QC
-from ..qc.utils import get_conflicting_tiles, get_num_pos_neg_tiles, get_mixed_subtype_tile_counts
-from ..subtype import Subtype
-from ..subtyping_params import SubtypingParams
+from biohansel.subtype.qc.const import QC
+from biohansel.subtype.qc.utils import get_conflicting_tiles, get_num_pos_neg_tiles, get_mixed_subtype_tile_counts
+from biohansel.subtype.subtype import Subtype
+from biohansel.subtype.subtyping_params import SubtypingParams
 
 
 def is_overall_coverage_low(st: Subtype, df: pd.DataFrame, p: SubtypingParams) -> Tuple[Optional[str], Optional[str]]:
```

Comment:
perhaps a different name for "p"?

Reply:
Changed in 9511966bb5c1825cc60001ba162d0f5045a8751a